L58-python-async-api.md

```diff
@@ -1660,6 +1660,11 @@ class StreamStreamClientInterceptor:
 
 ### Utility Functions
 
+```Python
+# grpc.aio.EOF is a unique object per process that evaluates to False
+(grpc.aio.EOF or False) == False
```

Comment:
I think it would be clearer to simply say `bool(grpc.aio.EOF) == False` or to simply list its implementation:

```python
class EOF:
    def __bool__(self):
        return False
```

Reply:
I hope there should only be one instance of `grpc.aio.EOF`, so the code you provided should be the actual implementation class named `_EOF`. And then assign to EOF: `EOF = _EOF()`. Or we can do some crazy meta class magic here.